src/core/service/notifications/jobs/handler.go

```diff
@@ -87,3 +91,13 @@ func (h *Handler) HandleReplication() {
 		return
 	}
 }
+
+// HandleReplicationTask handles the webhook of replication task
+func (h *Handler) HandleReplicationTask() {
+	log.Debugf("received replication task status update event: task-%d, status-%s", h.id, h.status)
+	if err := h.executionMgr.UpdateTaskStatus(h.id, h.status); err != nil {
```

Comment:
Also need to update the job id to the record

Reply:
no need to modify.